modules/logging-release-notes-5-7-10.adoc

```diff
@@ -0,0 +1,85 @@
+//module included in logging-5-7-release-notes.adoc
+
+:content-type: REFERENCE
+[id="logging-release-notes-5-7-10"]
+= Logging 5.7.10
+This release includes link:https://access.redhat.com/errata/RHSA-2024:0268[OpenShift Logging Bug Fix Release 5.7.10].
+
+[id="logging-release-notes-5-7-10-known-issue"]
+== Known issue
+Currently, the log alert is not available on an IPv6-enabled cluster. There is currently no workaround for this issue. (link:https://issues.redhat.com/browse/LOG-4891[LOG-4891])
```

Comment:
As mentioned elsewhere the updated text is on: https://issues.redhat.com/browse/LOG-4891